.github/workflows/unversioned_pages.yml

```diff
@@ -33,15 +38,22 @@ jobs:
           cp -Rv ./dev/release/ ./stable/
           rm -rf ./stable/roadmaps
           cp -Rv ./dev/roadmaps/ ./stable/
+          echo "Copy dev/index.html to stable folder"
           cp -v ./dev/index.html ./stable/
+          echo "In stable unversioned folder, find all regular files."
+          echo "If /_static is in file, substitute /../dev/_static by editing in place." 
           find stable/community/ -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
           find stable/developers/ -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
           find stable/naps/ -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
           find stable/release/ -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
           find stable/roadmaps/ -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
+          echo "Find _static and replace with ../dev/_static in the stable/index.html file."
           sed -i 's+_static+../dev/_static+g' stable/index.html
+          echo "Set STABLE environmment variable to the version found in the symlinked stable directory"
           STABLE=$(basename $(readlink -f ./stable))
+          echo "Updates the theme switcher version from 'dev' to stable numeric version '0.6.3' in all files of the stable directory."
           find stable/ -type f -exec sed -i "s+DOCUMENTATION_OPTIONS.theme_switcher_version_match = 'dev';+DOCUMENTATION_OPTIONS.theme_switcher_version_match = '${STABLE}';+g" {} +
+          echo "For all files in 'stable' directory, set show_version_warning_banner to false"
```

Comment:
We will need to revisit this (because this change is permanent, and is what is disabling the version warning banner permanently for these pages) but looks ok for now.